src/content/web/wasm.md

```diff
@@ -0,0 +1,74 @@
+---
+title: WebAssembly (Wasm) compilation
+description: Learn how to compile your Dart web app to WebAssembly.
+---
+
+The Dart team is excited to add
+[WebAssembly](https://webassembly.org/) as a compilation target when building
+Dart and [Flutter]({{site.flutter}}/wasm) applications for the web.
+
+Development of WebAssembly support remains ongoing,
+which will potentially result in frequent changes. 
+Revisit this page for the latest updates.
+
+:::note
+**Support for Wasm is now in beta!**
+
+WebAssembly support for Dart web is available on the Dart 
+*beta* and *main* [channels](/get-dart#release-channels).
+:::
+
+## WebAssembly support
+
+The current version of Dart compilation to WebAssembly has a number of
+restrictions:
+
+1. It targets WebAssembly with Garbage Collection (aka *WasmGC*), so...
+
+1. The compiled Wasm output currently targets JavaScript environments
+   (such as browsers), and thus currently doesn't support execution in standard
+   Wasm run-times like wasmtime and wasmer. For details, see
+   [issue #53884]({{site.repo.dart.sdk}}/issues/53884)
+
+1. Only Dart's
+   [next-gen JS interop mechanism](/interop/js-interop/)
+   is supported when compiling to Wasm.
+
+## Compiling your web app to Wasm {:#compiling-to-wasm}
+
+We've landed support in the `dart` CLI for invoking the
+`dart2wasm` compiler in the Dart and Flutter `main` channels:
```

Comment:
I know the engineers have been using the term `dart2wasm` but it's an implementation detail, and not aligned with how we talk about other compilers, so I think we should just call it the "Wasm compiler"

Reply:
Should I also remove the `main` mention from this sentence like you [mentioned](https://github.com/dart-lang/site-www/pull/5781#discussion_r1593745381) below?

Reply:
Yeah, I think we can remove the `main` part